lib/commands/deployproxy.js

```diff
@@ -719,6 +719,15 @@ function deployProxy(opts, request, done) {
     }, function(err, req, body) {
       if (err) { return done(err); }
 
+      // sometimes the body isn't json, sometimes, when things go wrong, it's xml
+      if ((body) && (body.trim().startsWith('<'))) {
```

Comment:
Any reason why you didn't use a `try/catch` around the `JSON.parse` a few lines lower?

Reply:
no, good call.